Rmarkdown/Oncology_Efficacy_Plots.Rmd

```diff
@@ -77,6 +103,31 @@ dose_record <- dose_record %>%
 
 dose_intensity <- dose_record[c("IDSHORT","COMB","ave_dose_intensity")]
 dose_intensity <- subset(dose_intensity, !duplicated(IDSHORT))
+
+#units and labels
+time_units_dataset     = "hours"
+time_units_plot        = "days"
+trtact_label           = "Dose"
+dose_label             = "Dose (mg)"
+time_label             = "Time(Days)"
+timemonth_label        = "Time (months)"
+sld_label              = "Percent Change in\nSum of Longest Diameters"
+sadose_label           = "ABC123 Dose (mg)"
+combodose_label        = "DEF456 Dose (mg)"
+bpchg_label            = "Best Percent Change \n in Tumor Size (%)"
+saauc_label            = "ABC123 AUC0-24 (ng.h/ml)"
+saaveauc_label         = "Average ABC123 dose (mg) \n up to the first assessment"
```

Comment:
"avg" is a more common abbreviation for average, my first thought when I see "ave" is "avenue"

Reply:
mono_avg_auc_label

Reply:
done!